src/core/internal/elf/dl.d

```diff
@@ -160,3 +213,20 @@ else
 {
     extern(C) const(char)* getprogname();
 }
+
+unittest
+{
+    import core.stdc.stdio;
+
+    char[512] buffer = void;
+    foreach (object; SharedObjects)
+    {
+        const name = object.name();
+        assert(name.length);
+        const path = object.getPath(buffer);
```

Comment:
I've originally asserted that `path` isn't null (no error), but that failed for some Linux CI systems for LDC. I haven't bothered looking into what the issue is.